desktop-widgets/printoptions.ui

```diff
@@ -93,6 +93,13 @@
         </property>
        </widget>
       </item>
+      <item>
+       <widget class="QCheckBox" name="printLandscape">
+        <property name="text">
+         <string>Page orientation landscape</string>
```

Comment:
This adds a new translation. What's the plan @dirkhh?

Reply:
Clearly I was completely thrown off course and it seems ridiculous to stop everything until I can get my act together.
Admittedly one of the cloud servers going away over night the other week didn't help.

Right now I'm just merging the things that are ready and I'll look into releasing a new version afterwards.